library/src/main/java/com/alamkanak/weekview/WeekView.java

```diff
@@ -179,6 +179,11 @@
     private ScrollListener mScrollListener;
     private AddEventClickListener mAddEventClickListener;
 
+    //Scroll by amount of days desired functionnality
+    private boolean mIsCustomScrollableAmountsOfDaysEnabled = false;
+    private float startOriginForScroll = 0;
+    private float sizeOfScrollableView;
```

Comment:
Don't create a field for this, but use a method.

Reply:
Ok